build/build/src/engine/context.rs

```diff
@@ -199,6 +199,26 @@ impl RunContext {
             };
             sbt.call_arg("syntax-rust-definition/Runtime/managedClasspath").await?;
         }
+        if self.config.build_native_runner {
+            env::ENSO_LAUNCHER.set(&engine::EngineLauncher::DebugNative)?;
+        } else {
+            env::ENSO_LAUNCHER.set(&engine::EngineLauncher::Shell)?;
```

Comment:
Shouldn't we specify nothing here? If the `ENSO_LAUNCHER` variable remains unset, then the build will pick default from `sbt` definitions - e.g. fallback to `shell` right now.

In the future it'd be enough to change the default in `sbt` definition and check `BuildInfo.isReleaseMode` which I believe _shouldn't be checked at all_ right now.

Reply:
You are right. Let's simplify things and not set the env var unless necessary. Done in 318c786a79d4f8b220180b45be11e278e3fa8c88